src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -2071,13 +2074,28 @@ messages.stmsg.m0139 = DOWNLOADED
 messages.stmsg.m0140 = Downloading...
 messages.stmsg.m0142 = Paused
 messages.stmsg.m0143 = DOWNLOAD
-messages.stmsg.desktop.contentCantDownload = This content can not be downloaded
+messages.stmsg.desktop.contentCantDownload = This item can’t be downloaded as of now
 frmelmnts.lbl.myEnrolledCollections = My Enrolled Collection
 frmelmnts.lbl.BatchExpiringIn = Batch expiring in
+frmelmnts.lbl.deleteConfirm = Are you sure you want to delete this Entity ?
```

Comment:
Some strings are in title case while some in Sentence Case. Please cross check once